src/quart/helpers.py

```diff
@@ -288,7 +288,10 @@ async def send_from_directory(
 
     See :func:`send_file` for the other arguments.
     """
-    file_path = Path(safe_join(str(directory), file_name))
+    raw_file_path = safe_join(str(directory), file_name)
+    if not raw_file_path:
```

Comment:
Could you change to `if raw_file_path is None`.

Reply:
done ;-)